extension/src/popup/views/SignTransaction/Preview/Data/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import "./styles.scss";
+import { CopyText, Icon } from "@stellar/design-system";
```

Comment:
NIT: `import { CopyText, Icon } from "@stellar/design-system";` should come before the style import. This should be throwing an eslint warning

Reply:
weird I need to figure out why that rule doesn't trigger for me. I get other lint warnings, and even other import order lint warnings but those didn't trigger for some reason.
Fixed in efda2d157b9925c274d733854f10634fea9e495d